app/src/main/kotlin/com/saltedge/authenticator/widget/passcode/KeypadView.kt

```diff
@@ -56,12 +57,15 @@ class KeypadView(context: Context, attrs: AttributeSet) : LinearLayout(context,
         when (view.id) {
             R.id.fingerView -> fingerprintActionClickListener?.onBiometricInputSelected()
             R.id.deleteView -> clickListener?.onDeleteKeyClick()
+            R.id.forgotView -> fingerprintActionClickListener?.onResetPasscode()
```

Comment:
fix naming.   
by name `fingerprintActionClickListener` - listens Fingerprint Actions.
and call of `onResetPasscode` looks strange.